rsconnect/main.py

```diff
@@ -475,7 +476,15 @@ def deploy_manifest(name, server, api_key, insecure, cacert, new, app_id, title,
         check_server_capabilities(connect_server, [is_conda_supported_on_server])
 
     with cli_feedback('Creating deployment bundle'):
-        bundle = make_manifest_bundle(file)
+        try:
+            bundle = make_manifest_bundle(file)
+        except IOError as error:
+            msg = 'Unable to include the file %s in the bundle: %s' % (error.filename, error.args[1])
+            if error.args[0] == errno.ENOENT:
+                msg = '\n'.join([msg, 'Since the file is missing but referenced in the manifest, you will need to\n'
```

Comment:
```suggestion
                msg = '\n'.join([msg, 'Because the file is missing but referenced in the manifest, you will need to\n'
```

Reply:
Optional: some are sticklers about [using because for cause and since for time](https://www.writersdigest.com/online-editor/since-vs-because) but it's not clear that "since" is incorrect so you can take this or leave it.